apps/web/app/components/container.tsx

```diff
@@ -0,0 +1,15 @@
+import { Flex, FlexProps } from 'leather-styles/jsx';
+
+type ContainerProps = FlexProps;
+
+export function Container(props: ContainerProps) {
+  return (
+    <Flex
+      flexDirection="column"
+      maxWidth="1216px"
+      mb="space.05"
+      mx={['space.04', 'space.04', 'space.05', 'space.05', 'auto']}
+      {...props}
+    />
+  );
+}
```

Comment:
What does this container do? We already have `Page`

Reply:
there was a little bit mess in local work, bc of previous fast copy pasting
removed